package.json

```diff
@@ -15,11 +15,10 @@
     "prepublishOnly": "yarn build && yarn lint && yarn test",
     "publish:all": "./scripts/publish-all.sh",
     "build": "yarn workspaces foreach --parallel --verbose run build",
-    "build:dev": "yarn workspaces foreach --parallel --verbose run build:dev",
     "build:clean": "yarn clean && yarn build",
     "clean": "yarn workspaces foreach --parallel --verbose run clean",
-    "serve": "echo 'Servers starting...' && $(yarn static packages/site/dist & yarn workspace @metamask/test-snap-confirm serve & yarn workspace @metamask/test-snap-error serve & yarn workspace @metamask/test-snap-bip44 serve & yarn workspace @metamask/test-snap-managestate serve & yarn workspace @metamask/test-snap-notification serve & yarn workspace @metamask/test-snap-bip32 serve)",
-    "lint:eslint": "eslint . --cache --ext js,ts",
+    "start": "yarn workspaces foreach --parallel --verbose --exclude website --exclude root run build && yarn workspaces foreach --parallel --verbose --interlaced --exclude root --jobs unlimited run start",
```

Comment:
Why doe we need two commands?

Reply:
One builds the Snaps, the other starts the servers.

Reply:
Why can't we do that in one command?